DirectX11Proj/systemclass.cpp

```diff
@@ -138,6 +143,8 @@ bool SystemClass::Frame()
 		return false;
 	}
 
+	mApplication->SceneTick();
+	mApplication->Tick();
```

Comment:
someone would guess that Tick would also call SceneTick - maybe update naming of either one